pom.xml

```diff
@@ -79,11 +79,11 @@
 	</dependencyManagement>
 
 	<properties>
-		<openMRSVersion>1.10.6</openMRSVersion>
-		<openMRSMinorVersion>1.9</openMRSMinorVersion>
-		<openMRSRuntimeVersion>1.9.12</openMRSRuntimeVersion>
+		<openMRSVersion>2.7.3</openMRSVersion>
+		<openMRSMinorVersion>2.7</openMRSMinorVersion>
+		<openMRSRuntimeVersion>2.7.3</openMRSRuntimeVersion>
```

Comment:
Can we get rid of these other openmrs version properties?  What is the point of them?